document deprecated variables and outputs feature (#1690)
hashicorp/web-unified-docs: content/terraform/v1.15.x (rc)/docs/language/block/module.mdx

`depends_on` is a **meta-argument**. Meta-arguments are built into the Terraform language and control how Terraform creates resources. Refer to the [`depends_on` reference](/terraform/language/meta-arguments/depends_on) for details about how the argument works.

### `ignore_nested_deprecations`

<Note>

Deprecated values are available in Terraform v1.15 and later.

</Note>

> If the `ignore_nested_deprecations` argument is set to true, Terraform won't show any deprecation warnings within the module call or nested modules.

Reviewer: ```suggestion
If the `ignore_nested_deprecations` argument is set to true, Terraform does not show deprecation warnings in the module call or nested modules.
```
